apps/payment/mixins.py

```diff
@@ -0,0 +1,83 @@
+from abc import abstractmethod
+from typing import List
+
+from django.conf import settings
+from django.utils import timezone
+
+from apps.authentication.models import OnlineUser as User
+
+
+class PaymentMixin:
```

Comment:
```suggestion
class PaymentMixin(ABC):
```
This is an abstract class

Reply:
I tried this out, and it just created a clusterfuck with the metaclasses in migrations. I'll leave it out for now.